examples/bevy_utilities/src/noise.rs

```diff
@@ -29,6 +29,42 @@ pub fn generate_noise_chunks2(
     })
 }
 
+pub fn generate_noise_chunk3(
+    chunk_min: Point3i,
+    chunk_shape: Point3i,
```

Comment:
These could just be `chunk_extent: Extent3i`.

Reply:
Done.